systest/Makefile

```diff
@@ -71,47 +72,50 @@ PROJDIR := $(shell dirname $(realpath $(MAKEFILE_DIR)))
 TAGINFO := $(shell git describe --long --tags --first-parent)
 
 # git branch
-BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+BRANCH := liberty
 
-# /ASB/PATH/systest/test_results: This is where the buildbot system will look
+# /ASB/PATH/systest/test_results: This is where the CI system will look
 # for test reuslts.
 RESULTS_DIR := $(MAKEFILE_DIR)/test_results
 TESTENVLOGDIR := $(RESULTS_DIR)/f5-openstack-agent_$(BRANCH)/testenv
 
 #### INITIALIZATION SECTION
-#    Since all tests will run on the same buildbot worker, and we do not (yet)
+#    Since all tests will run on the same CI worker, and we do not (yet)
 #    require different packages for different suites, all packages are
-#    installed.  Each deploy maps to its own GUMBALLS project.  This section
-#    has one because unittests are run in this section.  
+#    installed.  Each deploy maps to its own GUMBALLS project.
 #    Tests use tox for python variants.
-functest: 
+setup:
+	-@echo executing $@
+	sudo -E -H $(SCRIPT_DIR)/install_test_infra.sh
```

Comment:
I believe `-H` is redundant with `-E`, it's not necessarily the case that _this_ commit should modify that, but probably _a_ commit should excise `-H`.